snippet-template.md

```diff
@@ -0,0 +1,33 @@
+### ComponentName
+
+Explain briefly what the snippet does.
+
+Explain briefly how the snippet works.
+
+```jsx
+function ComponentName(props) {
+  return <div>{props}</div>;
+};
+```
+<!-- OR -->
+```jsx
+class ComponentName extends React.Component {
+  constructor(props){}
+  render(){
+    return <div>{props}</div>;
+  }
+}
+```
+
+```jsx
+ReactDOM.render(<ComponentName />, mountNode);
+```
+
+<!-- tags: (separate each by a comma) -->
+
+<!-- expertise: (0,1,2) -->
+<!-- Expertise levels:
+	0: beginner
+	1: intermediate
+	2: advanced
```

Comment:
We will want an expert level here. Eventually there will be a use case, even if it's archived, for render properties.